src/libs/actions/Report.js

```diff
@@ -950,6 +950,14 @@ function fetchAllReports(
         .then((returnedReportIDs) => {
             Onyx.set(ONYXKEYS.INITIAL_REPORT_DATA_LOADED, true);
 
+            // If at this point the user still doesn't have a Concierge report, create it for them.
+            // This means they were a participant in reports before their account was created (e.g. default rooms)
+            const hasConciergeChat = _.some(allReports, report => isConciergeChatReport(report));
+            if (!hasConciergeChat) {
+                // eslint-disable-next-line no-use-before-define
```

Comment:
I don't think we need this anymore based on the usage above that doesn't have it?